Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/bean/ConceptsMergeBean.java

```diff
@@ -0,0 +1,164 @@
+package edu.asu.conceptpower.app.bean;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import edu.asu.conceptpower.app.error.CPError;
+
+public class ConceptsMergeBean {
+
+    private String selectedPosValue;
+    private String selectedListName;
+    private String selectedTypeId;
+    private String selectedTypeName;
+    private String selectedConceptId;
+    private List<String> descriptions = new ArrayList<>();
+    private Set<String> synonymsids = new HashSet<>();
+    private Set<String> equalsValues = new HashSet<>();
+    private Set<String> similarValues = new HashSet<>();
+    private Set<String> mergedIds = new HashSet<>();
```

Comment:
MergedIds contains the details of what concepts are getting merged. 

SelectedConceptId contains the id to which the merged concept details needs to be stored. If the selectedConceptId is blank or null, we store the merged concept details in a new concept.

ConceptIds will contain only the CCP concept Id and not the wordnet concept id. This field is used for showing the user with the list of concepts to which the merged concept can be stored. So wordnet ids are removed from this list.